code/game/objects/items/devices/litanybook.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/device/litanybook
+
+	name = "\improper tribunal codex"
+	desc = "A holy text of the Moroz Holy Tribunal, the state religion of the Empire of Dominia."
+	desc_fluff = "An almost mandatory possession for any Dominian household, containing a list of Edicts, litanies, and rituals."
+	contained_sprite = TRUE
+	icon = 'icons/obj/human_items.dmi'
+	icon_state = "dominiabook"
+	item_state = "dominiabook"
+
+	drop_sound = 'sound/items/drop/cloth.ogg'
+
+	pickup_sound = 'sound/items/pickup/cloth.ogg'
+
+/obj/item/device/litanybook/attack_self(mob/user)
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+	user.visible_message(SPAN_NOTICE("You flip through [src]."))
+
+	var/q // recycled from tip of the day code. it just works!(TM)
+	var/quip // in actuality it randomly determines a value, then uses that value to select a particular line from the txt below
+	if(quip)
+		q = quip
+	else
+		var/list/randomquip = file2list("ingame_manuals/dominia.txt")
+		if(randomquip.len)
+			q = pick(randomquip)
+
+	if(do_after(user, 25))
+		to_chat(user, "<span class='notice'>You notice a particular verse: [q]</span>")
+
+/obj/item/device/litanybook/gadpathur
+
+	name = "\improper gadpathurian morale manual"
+	desc = "A popular Gadpathurian pocket guide, used to carry a fragment of the Commander's wisdom abroad."
+	desc_fluff = "Manufactured using recycled paper and canvas. The wisdom within the text is based on the collected speeches of Commander Patvardhan and her assorted councillors."
+	icon_state = "gadpathurbook"
+	item_state = "gadpathurbook"
+
+/obj/item/device/litanybook/gadpathur/attack_self(mob/user)
```

Comment:
This is some duplicated code that can likely be avoided. Also, why can't this just be a regular book/manual anyway?

Reply:
I'll see what I can trim without mixing contents- as for 'why can't it be a regular book', the book functionality is a nightmare. These are meant to be more of a 'quick fix' of relevant thought rather than a continuous read. I'll see what I can do, though!